api/db.py

```diff
@@ -56,6 +56,29 @@ def fetchSingleEdition(self, editionID, showAll=False):
             )\
             .filter(Edition.id == editionID).first()
 
+    def fetchSortedEditions(self, sortField: str, sortDirection: str, limit: int):
+        sortClause = {
+            "title": Edition.title,
+            "date": Edition.date_created,
+        }.get(sortField)
+        if not sortClause:
+            raise ValueError(f"Invalid sort param {sortField}")
+        if sortDirection == "DESC":
+            sortClause = sortClause.desc()
+
+        return (
+            self.session.query(Edition)
```

Comment:
I have one concern here, which is that this could/will return many editions for a single work in the collection (since they all get created together). I _think_ what we want is really to get the `Work` record and then send back what we would generally call the "preferred" edition. That is a bit more complicated but not too bad?

Reply:
Latest commit should address that using some window function magic!